tox.ini

```diff
@@ -0,0 +1,60 @@
+[tox]
+skipsdist=True
+envlist=
+    lint,
+    py{27,36}-dj{18,111}-wag{113},
+    py{36}-dj{20}-wag{20}
+
+[testenv]
+install_command=pip install -e ".[testing]" -U {opts} {packages}
+commands=
+    coverage erase
+    coverage run --source='treemodeladmin' {envbindir}/django-admin.py test {posargs}
```

Comment:
How would you feel about adding a third `coverage report` line here? I've been thinking lately that this would be a good pattern to adopt in our packages, and I think I've seen @higs4281 use this. An advantage of including the report as part of the tox execution is that then you don't need to have coverage installed outside of the tox env to view the reports; a downside is for tox setups with many environments you might see the report many times (although, maybe this is good to ensure that the coverage is consistent across versions).

Reply:
I actually like this idea a lot. I'll do it!